tests/integration/workflows/test_workflows_tasks_actions.py

```diff
@@ -453,5 +433,5 @@ def test_refextract_from_pdf(
     citing_doc_eng = WorkflowEngine.from_uuid(citing_doc_workflow_uuid)
     citing_doc_obj = citing_doc_eng.processed_objects[0]
 
-    assert citing_doc_obj.data['references'][7]['record']['$ref'] == 'http://localhost:5000/api/literature/1'
+    assert citing_doc_obj.data['references'][7]['record']['$ref'] == 'http://localhost:5000/api/literature/1000'
```

Comment:
Nit: I don't understand this change... is there some state leaking from a previous workflow test?

Reply:
I doubt that but there does seem to be some issue with multiple records being inserted in a single test. So if I insert the cited record, and then try to run the workflow for the citing record, I get some error that the PID already exists. This is possibly some artefact of how we insert records using TestRecordMetadata.

Reply:
In fact, I am sure that it's not a leak from other test since this happens even if I just run onlt this particular test.

Reply:
I see. Maybe the workflow will try to insert record 1 because that's the state of the PID counter, but the test fixture already put there a record 1 without updating the PID counter.

Let's keep the code like this for now, but this is a little too brittle for my taste...